Assets/Mirror/Runtime/NetworkManager.cs

```diff
@@ -24,45 +24,53 @@ public enum NetworkManagerMode { Offline, ServerOnly, ClientOnly, Host }
     [HelpURL("https://mirror-networking.com/docs/Components/NetworkManager.html")]
     public class NetworkManager : MonoBehaviour
     {
+        #region Inspector Fields
```

Comment:
the improvements look good, but a region for public vars is just too much.

Reply:
Removed